pkg/commands/server/pluginshealth_test.go

```diff
@@ -136,3 +136,102 @@ func TestPluginsHealthCommandRequestSuccessJson(t *testing.T) {
 	assert.Assert(t, golden.String(app.OutBuffer.String(), "pluginshealth.success.json.golden"))
 	test.ExpectNoError(t, err)
 }
+
+// TestPluginsHealthCommandExtraArgsRequestSuccessYaml tests the 'plugins health'
+// command using a [PLUGIN TAG] argument when extra arguments are provided.
+func TestPluginsHealthCommandExtraArgsRequestSuccessYaml(t *testing.T) {
+	test.Setup()
+
+	app := test.NewFakeApp()
+	app.Commands = append(app.Commands, ServerCommand)
+
+	err := app.Run([]string{
+		app.Name,
+		"--format", "yaml",
+		ServerCommand.Name,
+		pluginsCommand.Name,
+		pluginsHealthCommand.Name,
+		"vaporio/emulator-plugin", "extra",
+	})
+
+	t.Logf("Standard Out: \n%s", app.OutBuffer.String())
+	t.Logf("Standard Error: \n%s", app.ErrBuffer.String())
+
+	// FIXME: Including the error "command 'health' requires between 0 and 1
+	// arguments, 2 given" in the golden file makes the test fail? It is
+	// currently empty.
+	assert.Assert(t, golden.String(app.OutBuffer.String(), "pluginshealth.error.extra_args.golden"))
```

Comment:
RE: the FIXME, 

the reason that is happening is because in the assertion, you are testing `app.OutBuffer`, but the error is written to  `app.ErrBuffer`, so it should work if you change `OutBuffer` to `ErrBuffer` -- same for the plugin info test case

Reply:
That makes sense.